REPLesent.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+REPLesent=$PWD/replesent.scala
+MSG="[REPLesent] -"
+
+if [ $# -eq 0 ]
+  then
+    echo "$MSG You need to supply the presentation file name as an argument!"
+    echo ""
+    echo "usage: $0 [yourpresentation.txt]"
+    exit 1;
+fi
+
+# Check that input is not absolute or relative path
+if [[ $1 == *"/"* ]]
+    then
+        echo "$MSG You can't use a relative or absolute path to locate the presentation!"
+        echo "$MSG Move first to the presentation directory and then call REPLesent from there."
+        exit 1;
+fi
+
+# Check that file exists in WD
+if [ ! -f $PWD/$1 ]
+    then
+        echo "$MSG No such file $1 in $PWD"
+        exit 1;
+fi
+
+id=$RANDOM-$RANDOM
+tmpfile="/tmp/$1-$id.scala"
+
+echo "val replesent = REPLesent(input=\"$1\", intp=\$intp);import replesent._;f;" >> $tmpfile
+
+echo -e "$MSG Starting presentation $1...\n"
+
+scala -Dscala.color -language:_ -nowarn -i $REPLesent -i $tmpfile
```

Comment:
You are just shifting the problem and adding one more
- You still need to provide a shell script in order to quick boot REPLesent, otherwise you need to first put your presentation in the same folder than the script, then cd and boot from there, which is inconvenient. With this script, it doesn't matter where the script. We should also probably fix the input parameter so that we can pass a path rather than a file.
- Now you need to load SBT first before the presentation and from there call the console which introduces unnecessary loading time to a process that already takes too much time. SBT was not meant to be used like this.

I thought I would make this pull request because I wrote this for myself and frankly it works quite neatly.